quixstreams/sinks/influxdb_v3.py

```diff
@@ -0,0 +1,140 @@
+            By default, the record timestamp will be used with "ms" time precision.
+            When using a custom key, you may need to adjust the `time_precision` setting
+            to match.
+        :param include_metadata_tags: if True, includes record's key, topic,
+            and partition as tags.
+            Default - `False`.
+        :param batch_size: how many records to write to InfluxDB at once.
+            Note that it only affects the size of the writing batch, and not the number
+            of records flushed on each checkpoint.
+            Default - `1000`.
+        :param enable_gzip: if True, enables gzip compression for writes.
+            Default - `True`.
+        :param request_timeout_ms: an HTTP request timeout in milliseconds.
+            Default - `10000`.
+        :param debug: if True, print debug logs from InfluxDB client.
+            Default - `False`.
+        """
+
+        super().__init__()
+        self._client = InfluxDBClient3(
+            token=token,
+            host=host,
+            org=organization_id,
+            database=database,
+            debug=debug,
+            enable_gzip=enable_gzip,
+            timeout=request_timeout_ms,
+            write_client_options={
+                "write_options": WriteOptions(
+                    write_type=WriteType.synchronous,
+                )
+            },
+        )
+        self._measurement = measurement
+        self._fields_keys = fields_keys
+        self._tags_keys = tags_keys or []
+        self._include_metadata_tags = include_metadata_tags
+        self._time_key = time_key
+        self._write_precision = time_precision
+        self._batch_size = batch_size
+
+    def write(self, batch: SinkBatch):
+        measurement = self._measurement
+        fields_keys = self._fields_keys
+        tags_keys = self._tags_keys
+        time_key = self._time_key
+        for write_batch in batch.iter_chunks(n=self._batch_size):
+            records = []
+            for item in write_batch:
+                value = item.value
+                tags = {tag_key: value[tag_key] for tag_key in tags_keys}
+                if self._include_metadata_tags:
+                    tags["__key"] = item.key
+                    tags["__topic"] = batch.topic
+                    tags["__partition"] = batch.partition
+                fields = (
+                    {field_key: value[field_key] for field_key in fields_keys}
+                    if fields_keys
+                    else value
+                )
+                ts = value[time_key] if time_key is not None else item.timestamp
+                record = {
+                    "measurement": measurement,
+                    "tags": tags,
+                    "fields": fields,
+                    "time": ts,
+                }
+                records.append(record)
+            try:
+                self._client.write(
```

Comment:
I am missing any sort of logging here. It looks like sink is not working because of that.